static/app/views/organizationStats/usageStatsProjects.tsx

```diff
@@ -19,6 +20,8 @@ import withProjects from 'sentry/utils/withProjects';
 import {UsageSeries} from './types';
 import UsageTable, {CellProject, CellStat, TableStat} from './usageTable';
 
+type Router = InjectedRouter | null | undefined;
```

Comment:
instead of passing the Router through multiple children components to get it where you want to be, i think you could use `RouteComponentProps`

Reply:
fixed!